src/styles/_header.scss

```diff
@@ -37,5 +37,11 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+
+    transition: color 0.3s;
+  }
+
+  &__link:hover {
+    color: #00acdc;
   }
```

Comment:
Use nesting here more details in  [video](https://mate.academy/learn/css-advanced/sass#/video/925)